pkg/mcproducer/mcproducer.go

```diff
@@ -75,14 +91,11 @@ func executeIntoBase64(tmpl *template.Template, params map[string]any) (string,
 	return buf.String(), nil
 }
 
-func verifyImageFormat(image string) error {
-	_, digestErr := name.NewDigest(image)
-	if digestErr == nil {
-		return nil
-	}
-	_, tagErr := name.NewTag(image)
-	if tagErr == nil {
-		return nil
+func getLocalFileName(containerImage string) (string, error) {
+	_, err := name.ParseReference(containerImage)
+	if err != nil {
+		return "", fmt.Errorf("failed to parse container image %s name: %v", containerImage, err)
 	}
-	return fmt.Errorf("invalid image %s, input should be either in tag or digest format. Digest error %v, Tag error %v", image, digestErr, tagErr)
+
+	return "/var/lib/image_file_day1.tar", nil
```

Comment:
This restricts the day1 solution to one image only. Should we make that path unique? See #920.

Reply:
I think that a better solution would be to use soft dependencies: saves loading different MCOs, creating new additional services and pulling different images